examples/websocket-basic-sample/websocket-basic-sample.bal

```diff
@@ -37,30 +37,33 @@ service<http:WebSocketService> SimpleSecureServer bind ep {
             conn -> ping(pingData) but {error e => log:printErrorCause("Error sending ping", e)};
         } else if (text == "closeMe") {
             conn -> close(1001, "You asked me to close the connection")
-                         but {error e => log:printErrorCause("Error occured when closing the connection", e)};
+                         but {error e => log:printErrorCause("Error occurred when closing the connection", e)};
         } else {
-            conn -> pushText("You said: " + text) but {error e => log:printErrorCause("Error occured when sending text", e)};
+            conn -> pushText("You said: " + text) but {error e => log:printErrorCause("Error occurred when sending
+            text", e)};
```

Comment:
This new line is in the middle of text. Can we remove it?

Reply:
Ack! Sent a new commit with the changes.